### PR TITLE
feat: removed NgbTabsetModule and ngbAutofocus #2134 #2133

### LINKED ISSUE #2133: Remove ngbAutofocus directive
Suggested-addresses-dialog and add-to-cart component uses `ngbAutofocus` in the html and `NgbActiveModal` to render the component. This component is the modal for the address suggestion. This ticket should:

- Remove the use of `ngbAutofocus`
- Implement custom logic and styling mimicking `ngbAutofocus`
- **Keep** the use of `NgbActiveModal`, this will be removed in a subsequent ticket

Don't forget to adjust the unit test and remove the BootstrapModule import from this component's module.

### LINKED ISSUE #2134: Remove ngBootsrap store-finder-list-item component
Store-finder-list-item component imports `NgbTabsetModule` in its unit test but not in its component implementation . This ticket should:

- Remove the use of `NgbTabsetModule` in the spec file
- Make sure it's not used in the component code

### PR DESCRIPTION
**Pull request by Pucek9**
Closes #2134
Closes #2133

### CODE REVIEW

File: projects/storefrontlib/src/shared/directives/auto-focus/auto-focus.directive.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { AfterViewInit, Directive, ElementRef } from '@angular/core';
+
+@Directive({
+  selector: '[cxAutoFocus]',
+})
+export class AutoFocusDirective implements AfterViewInit {
```
Comment (Pucek9) [inline projects/storefrontlib/src/shared/directives/auto-focus/auto-focus.directive.ts]: which one is better here? AfterViewInit or AfterViewChecked?

File: projects/storefrontlib/src/shared/directives/auto-focus/auto-focus.directive.spec.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import { TestBed, ComponentFixture } from '@angular/core/testing';
+import { Component } from '@angular/core';
+import { By } from '@angular/platform-browser';
+import { AutoFocusDirective } from './auto-focus.directive';
+@Component({
+  template: `
+    <input type="text" cxAutoFocus value="5" />
```
[the remark this answers is not in the document]
Comment (Pucek9) [reply projects/storefrontlib/src/shared/directives/auto-focus/auto-focus.directive.spec.ts]: It's test only, but as you wish 😁 let me use string mockValue

File: projects/storefrontlib/src/shared/directives/auto-focus/auto-focus.directive.ts (hunk 1 above)
Comment (Pucek9) [reply projects/storefrontlib/src/shared/directives/auto-focus/auto-focus.directive.ts]: So in conclusion can we leave it as is?

Comment (Zeyber) [reply projects/storefrontlib/src/shared/directives/auto-focus/auto-focus.directive.ts]: Yup

**Pull request merged**